metagraph/src/cli/query.cpp

```diff
@@ -31,7 +34,24 @@ using namespace mtg::annot::binmat;
 using namespace mtg::graph;
 
 using mtg::common::logger;
-
+using mtg::graph::boss::BOSS;
+using mtg::graph::boss::BOSSConstructor;
+typedef typename mtg::graph::DeBruijnGraph::node_index node_index;
+
+
+QueryExecutor::QueryExecutor(const Config &config,
+                             const graph::AnnotatedDBG &anno_graph,
+                             const graph::align::DBGAlignerConfig *aligner_config,
+                             ThreadPool &thread_pool)
+      : config_(config),
+        anno_graph_(anno_graph),
+        aligner_config_(aligner_config
+                        ? new graph::align::DBGAlignerConfig(*aligner_config)
+                        : nullptr),
+        thread_pool_(thread_pool) {
+    if (aligner_config_)
+        aligner_config_->forward_and_reverse_complement = false;
```

Comment:
There is a mode in the `align` regime where only the best alignment of the forward and reverse complement is reported (it can potentially speed things up because the best alignment score for the forward alignment is used as a lower bound for the reverse complement alignment), but this is kind of a legacy feature since it's not relevant for canonical graphs or `CanonicalDBG` graphs. Maybe I should just remove it...

Reply:
Got it, it would have been clear if this there was an assertion here, so we expect that the config is compatible to the logic in this part.